applications/swbatch/app.json

```diff
@@ -0,0 +1,219 @@
+{
+    "id": "swbatch",
+    "version": "0.4.1",
+    "description": "swbatch v0.4.1 executes on a single node of the Frontera cluster. swbatch v0.4.1 uses geopsy v3.4.2 as its inversion engine, therefore all .target and .param inversion input files must be created with a compatible version of geopsy. The best way to ensure compatible input files is to use the example inversion workflow provided with the latest version of the open-source Python package swprepost.",
+    "owner": "${apiUserId}",
+    "runtime": "ZIP",
+    "runtimeVersion": null,
+    "runtimeOptions": null,
+    "containerImage": "tapis://cloud.data/corral/tacc/aci/CEP/applications/v3/swbatch/swbatch.zip",
+    "jobType": "BATCH",
+    "maxJobs": -1,
+    "maxJobsPerUser": -1,
+    "strictFileInputs": true,
+    "jobAttributes": {
+        "description": "",
+        "dynamicExecSystem": false,
+        "execSystemConstraints": null,
+        "execSystemId": "frontera",
+        "execSystemExecDir": "${JobWorkingDir}",
+        "execSystemInputDir": "${JobWorkingDir}",
+        "execSystemOutputDir": "${JobWorkingDir}/output",
```

Comment:
Something we've recently discovered is that outputs need to be referenced from `"${JobWorkingDir}"` unless you know that they will go to a specific subdirectory called `output`, which could be controlled from the app run script. Do we know where outputs go or what they look like?

Reply:
i removed /output